packages/hoprd/package.json

```diff
@@ -20,7 +20,7 @@
   "scripts": {
     "clean": "rimraf ./lib",
     "build": "yarn clean && tsc -p . && yarn buildAdmin && chmod u+x lib/*.js",
-    "buildAdmin": "next build hopr-admin",
+    "buildAdmin": "HOPRD_GIT_COMMIT=$(git rev-parse --short HEAD) && next build hopr-admin",
```

Comment:
Why are you using `&&` in this case instead of just running the command after assigning the env var?

Reply:
Nice catch, let me change it